examples/EvaluatingChatAssistant/EvaluatingChatAssistant.ipynb

```diff
@@ -0,0 +1,353 @@
+    "\n",
+    "Comment out or delete the line `...chat_history` from the `client.chat.completions.create(...)` request in the task function."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "const response = await client.chat.completions.create({\n",
+    "  model: \"gpt-4o\",\n",
+    "  messages: [\n",
+    "    {\n",
+    "      role: \"system\",\n",
+    "      content: \"You are a helpful and polite assistant who knows about sports.\"\n",
+    "    },\n",
+    "    // ...chat_history,\n",
+    "    {\n",
+    "      role: \"user\",\n",
+    "      content: input,\n",
+    "    }\n",
+    "  ]\n",
+    "});"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Run the eval again in a new `gpt-4o assistant - no history` experiment."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "Eval(\"Chat assistant\", {\n",
+    "  experimentName: \"gpt-4o assistant - no history\",\n",
+    "  data: () => dataset,\n",
+    "  task: runTask,\n",
+    "  scores: [Factuality],\n",
+    "  metadata: {\n",
+    "    model: \"gpt-4o\",\n",
+    "    prompt: \"You are a helpful, polite assistant who knows about sports.\"\n",
+    "  }\n",
+    "});"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "Experiment gpt - 4o assistant - no history is running at http://www.braintrust.dev/app/braintrustdata.com/p/Chat%20assistant/experiments/gpt-4o%20assistant%20-%20no%20history\n",
+    " ████████████████████████████████████████ | Chat assistant[experimentName = gpt - 4o... | 100 % | 5 / 5 datapoints\n",
+    "\n",
+    "=========================SUMMARY=========================\n",
+    "gpt-4o assistant - no history compared to gpt-4o assistant:\n",
+    "4.00% (-60.00%) 'Factuality' score      (0 improvements, 5 regressions)\n",
+    "\n",
+    "4.14s 'duration'        (2 improvements, 3 regressions)\n",
+    "\n",
+    "See results for gpt-4o assistant - no history at http://www.braintrust.dev/app/braintrustdata.com/p/Chat%20assistant/experiments/gpt-4o%20assistant%20-%20no%20history"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As we were expecting, the Factuality score went way down, from 64% to just 4%.\n",
+    "\n",
+    "If we open [the dashboard](https://www.braintrust.dev), we can compare the results of the two experiments\n",
+    "\n",
+    "![dist-chart](./assets/assistant-dist-comp.png)\n",
+    "\n",
+    "For one datapoint the model was able to glean some context from the user's input and provide some information that was factually consistent with the expected answer, but for the majority the model needed the context and specificity that the `chat_history` provides."
```

Comment:
Would be nice to show a trace view here of a row or something to show a specific output and score